pkg/gateway/model/app.go

```diff
@@ -19,6 +21,11 @@ const (
 	AssetGearSubdomain GearSubdomain = "assets"
 )
 
+var PathPrefixToGear = map[string]Gear{
+	"/_auth":  AuthGear,
```

Comment:
I think a slash should be appended to the path prefixes, so that `/_auth123/` would not be matched as auth gear.